Detailed Design Power system - Team 5_Spring 2026

Austin24 opened the pull request.

Austin24 committed:
Added info to sections

Austin24 committed:
Detailed_design

Austin24 committed:
update

Austin24 committed:
added power budgeting

Austin24 committed:
update to typo

Austin24 committed:
regex TABLE links

Austin24 committed:
Edits to document 240-120 conversion

Austin24 committed:
Quality of life edits plus component changes.

csjohnson43 reviewed (changes requested):
- You have discussed a 5 V logic rail with a flyback while the BOM implements a 2.9 V regulator and AC-DC modules. Double check this
- How will you test the system to validate power, thermal, or standards compliance?

Austin24 committed:
Edit to link to correct component for 5v regulator (U6)

Austin24 committed:
Edits to Power System (2/6/2026)

This edit consisted of changing words to fit the new design version. Edits to the schematic images. changes to the BOM. And an added section in Analysis titled "Testing".

Austin24 committed:
Added fuses to BOM for holder on PCB

Austin24 committed:
Update Detailed Design-Power System.md

Austin24 committed:
Final touches

Austin24 committed:
final touches 2.0

Austin24 committed:
Moved files to Power System Folder
